.eslintrc.yml

```diff
@@ -0,0 +1,5 @@
+extends: 'eslint-config-cheminfo'
+parserOptions:
+  sourceType: module
+env:
+  jest: true
```

Comment:
please remove the env property. Are you using an outdated version of the cheminfo generator? It was fixed a month ago

Reply:
cheminfo generator is not working alone with me, so I was using directly the yeoman generator, but it is indeed an outdated one